### ISSUE TITLE
LSP: bad argument to math.min in M.fancy_floating_markdown

### ISSUE THREAD

**Issue by vrischmann** [neovim/neovim#13067]
- `nvim --version`:

```
NVIM v0.5.0-dev
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -Wp,-U_FORTIFY_SOURCE -Wp,-D_FORTIFY_SOURCE=1 -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/builddir/build/BUILD/neovim-0.4.4.0.git.14720.a260d5def/x86_64-redhat-linux-gnu/config -I/builddir/build/BUILD/neovim-0.4.4.0.git.14720.a260d5def/src -I/usr/include -I/usr/include/lua-5.1 -I/builddir/build/BUILD/neovim-0.4.4.0.git.14720.a260d5def/x86_64-redhat-linux-gnu/src/nvim/auto -I/builddir/build/BUILD/neovim-0.4.4.0.git.14720.a260d5def/x86_64-redhat-linux-gnu/include
Compiled by mockbuild

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

- language server name/version:

gopls:
```
golang.org/x/tools/gopls master
    golang.org/x/tools/gopls@(devel)
```

- Operating system/version:

Fedora 32

<details>
<summary>nvim -c ":checkhealth nvim nvim_lsp"</summary>

```
health#nvim#check
========================================================================
## Configuration
  - OK: no issues found

## Performance
  - OK: Build type: RelWithDebInfo

## Remote Plugins
  - OK: Up to date

## terminal
  - INFO: key_backspace (kbs) terminfo entry: key_backspace=\177
  - INFO: key_dc (kdch1) terminfo entry: key_dc=\E[3~
  - INFO: $COLORTERM='truecolor'

## tmux
  - OK: escape-time: 50
  - INFO: Checking stuff
  - OK: focus-events: on
  - INFO: $TERM: screen-256color

health#nvim_lsp#check
========================================================================
## Checking language server protocol configuration
  - INFO: gopls: configuration checked.
  - INFO: zls: configuration checked.
```

</details>

<details>
<summary>lsp.log</summary>

```
[ DEBUG ] 2020-10-07T19:23:12+0200 ] /usr/share/nvim/runtime/lua/vim/lsp.lua:640 ]      "LSP[gopls]"    "client.request"        1       "textDocument/hover"    {  position = {    character = 18,    line = 14  },  textDocument = {    uri = "file:///home/vincent/dev/foobar/main.go"  }}     <function 1>    1
[ DEBUG ] 2020-10-07T19:23:12+0200 ] /usr/share/nvim/runtime/lua/vim/lsp/rpc.lua:361 ]  "rpc.send.payload"      {  id = 3,  jsonrpc = "2.0",  method = "textDocument/hover",  params = {    position = {      character = 18,      line = 14    },    textDocument = {      uri = "file:///home/vincent/dev/foobar/main.go"    }  }}
[ DEBUG ] 2020-10-07T19:23:12+0200 ] /usr/share/nvim/runtime/lua/vim/lsp/rpc.lua:462 ]  "decoded"       {  id = 3,  jsonrpc = "2.0",  result = {    contents = {      kind = "markdown",      value = "```go\nfunc csv.NewReader(r io.Reader) *csv.Reader\n```\n\n[`csv.NewReader` on pkg.go.dev](https://pkg.go.dev/encoding/csv#NewReader)\n\nNewReader returns a new Reader that reads from r\\.\n"    },    range = {      end = {        character = 20,        line = 14      },      start = {        character = 11,        line = 14      }    }  }}
[ DEBUG ] 2020-10-07T19:23:12+0200 ] /usr/share/nvim/runtime/lua/vim/lsp/callbacks.lua:332 ]    "default_callback"      "textDocument/hover"    {  bufnr = 1,  client_id = 1,  params = {    contents = {      kind = "markdown",      value = "```go\nfunc csv.NewReader(r io.Reader) *csv.Reader\n```\n\n[`csv.NewReader` on pkg.go.dev](https://pkg.go.dev/encoding/csv#NewReader)\n\nNewReader returns a new Reader that reads from r\\.\n"    },    range = {      end = {        character = 20,        line = 14      },      start = {        character = 11,        line = 14      }    }  }}
```

</details>

### Steps to reproduce using `nvim -u NORC`

The minimal configuration that reproduces the problem:
```
packadd lsp

set nowrap

lua << EOF
    require'nvim_lsp'.gopls.setup{}
EOF

let mapleader=","
au FileType go nmap <Leader>h  <cmd>lua vim.lsp.buf.hover()<CR>
```

### Actual behaviour

When calling `vim.lsp.buf.hover()` it fails with this:
```
Error executing vim.schedule lua callback: /usr/share/nvim/runtime/lua/vim/lsp/util.lua:811: bad argument #2 to 'min' (number expected, got boolean)
```

This is caused by `set nowrap`: if I remove it it works fine.

### Expected behaviour

The call to `vim.lsp.buf.hover()` should work even with `set nowrap`.

**Comment by vrischmann** [comment]
There's already #13043 which fixes this.

**Comment by mfussenegger** [comment]
Given that https://github.com/neovim/neovim/pull/13043 got merged,  can this be closed?

**Comment by vrischmann** [comment]
This is fixed for me with `NVIM v0.5.0-772-g2786d96fa`, I'll close this issue.
